packages/vscode-extension/syntaxes/teamsfx-toolkit-output.tmLanguage

```diff
@@ -106,7 +106,7 @@
                     </dict>
                 </dict>
                 <key>match</key>
-                <string>(https?:\/\/[a-zA-Z0-9_\-\/\.]*)</string>
+                <string>(https?:\/\/[^\s]*)</string>
```

Comment:
Consider using a stricter pattern to avoid possible corner cases.

Reply:
Thanks, updated.